tools/generate-changelog.py

```diff
@@ -181,7 +186,10 @@ def main():
     new_manifest = args.new
     docs_dir = args.docs
 
-    changes = '[ Changes in primed packages ]\n\n'
+    # add a header that helps us audit where the current build is
+    # sourced from.
+    changes = f"Built from commit: https://github.com/canonical/core-base/tree/{git_commit_hash()}\n\n"
```

Comment:
Should we hardcode this URL or should we try to do some kind of magic with the ouput of `git remote get-url origin` to autogenerate it?